importer/importer/services/fhir_keycloak_api.py

```diff
@@ -97,11 +97,10 @@ def __init__(self, options: FhirKeycloakApiOptions):
     )
     def request(self, **kwargs):
         # TODO - spread headers into kwargs.
-        headers = {"content-type": "application/json", "accept": "application/json"}
-        response = self.api_service.oauth.request(headers=headers, **kwargs)
+        response = self.api_service.oauth.request( **kwargs)
```

Comment:
The headers are no longer required? I they aren't I think we should get rid of the `TODO` comment as well

Reply:
I moved everything here https://github.com/onaio/fhir-tooling/blob/82d66083a66331fb8ae70c43a501aff242a81285/importer/importer/request.py#L12-L27